packages/fluentui/docs/src/routes.tsx

```diff
@@ -51,74 +51,77 @@ import EditorToolbarPrototype from './prototypes/EditorToolbar';
-          <Route exact path="/prototype-search-page" component={SearchPagePrototype} />
-          <Route exact path="/prototype-mentions" component={MentionsPrototype} />
-          <Route exact path="/prototype-dropdowns" component={DropdownsPrototype} />
-          <Route exact path="/prototype-popups" component={PopupsPrototype} />
-          <Route exact path="/prototype-alerts" component={AlertsPrototype} />
-          <Route exact path="/prototype-editor-toolbar" component={EditorToolbarPrototype} />
-          <Route exact path="/prototype-hexagonal-avatar" component={HexagonalAvatarPrototype} />
-          <Route exact path="/prototype-table" component={TablePrototype} />
-          <Route exact path="/prototype-nested-popups-and-dialogs" component={NestedPopupsAndDialogsPrototype} />
-          <Route exact path="/virtualized-tree" component={VirtualizedTreePrototype} />
-          <Route exact path="/virtualized-table" component={VirtualizedTablePrototype} />
-          <Route exact path="/prototype-copy-to-clipboard" component={CopyToClipboardPrototype} />
-          <Route exact path="/faq" component={FAQ} />
-          <Route exact path="/accessibility" component={Accessibility} />
-          <Route exact path="/accessibility-behaviors" component={AccessibilityBehaviors} />
-          <Route exact path="/focus-zone" component={FocusZone} />
-          <Route exact path="/focus-trap-zone" component={FocusTrapZone} />
-          <Route exact path="/auto-focus-zone" component={AutoFocusZone} />
-          <Route exact path="/theming" component={Theming} />
-          <Route exact path="/theming-examples" component={ThemingExamples} />
-          <Route exact path="/layout">
-            <MarkdownPage page={Layout} />
-          </Route>
-          <Route exact path="/shorthand-props">
-            <MarkdownPage page={ShorthandProps} />
-          </Route>
-          <Route exact path="/component-architecture">
-            <MarkdownPage page={ComponentArchitecture} />
-          </Route>
-          <Route exact path="/theming-specification">
-            <MarkdownPage page={ThemingSpecification} />
-          </Route>
-          <Route exact path="/integrate-custom-components" component={IntegrateCustomComponents} />
-          <Route exact path="/performance" component={Performance} />
-          <Route exact path="/composition">
-            <MarkdownPage page={Composition} />
-          </Route>
-          <Route exact path="/colors" component={Colors} />
-          <Route exact path="/color-palette" component={ColorPalette} />
-          <Route exact path="/color-palette-category" component={CategoryColorPalette} />
-          <Route exact path="/color-schemes" component={ColorSchemes} />
-          <Route exact path="/color-schemes-category" component={CategoryColorSchemes} />
-          <Route exact path="/*" component={PageNotFound} />
-        </Switch>
+        <PerfDataProvider>
```

Comment:
Is there any sence to fetch this data for prototypes and behaviors? Can we fetch it only for components pages?

Reply:
I don't see any way how to do that.
Once the docsite renders (`DocsLayout` is rendered) I want the data to be fetched and held for the whole lifetime of the docsite.

I agree it makes no sense to fetch the data if you go directly to a prototype (so I could theoretically start the fetch only after a chart is rendered) but I don't see this as the main problem. The problem this PR is solving is to avoid screener to kill the backend.